config/initializers/assets.rb

```diff
@@ -2,10 +2,10 @@
 
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
-
+Rails.application.config.assets.precompile += %w( pro.png )
```

Comment:
Can we add the regex to store all image files. So we don't have to add this manually for each file?